DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp

```diff
@@ -271,8 +271,8 @@ int main(int argc, char** argv) {
     // Instantiate multiple versions of the kernel
     // The II achieved by the compiler can differ between FPGA architectures
     //
-    // On Arria 10, we are able to achieve an II of 1 for all versions of the
-    // kernel.
+    // On Arria® 10, we are able to achieve an II of 1 for versions 1 and 2 of
+    // the kernel (not version 0).
```

Comment:
This does not seem like a valid change.  I suspect you have accidentally reverted to an older version of this file?

Reply:
Yes, will fix that now